ghcup-0.0.7.yaml

```diff
@@ -3494,6 +3494,100 @@ ghcupDownloads:
+              dlHash: 52a20a1f87b84ce6b34d830a849cd29404228cfbfeb114f81ea28bcb60edeca8
+            '(>= 10 && < 11)': &ghc-947-64-deb10
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-deb10-linux.tar.xz
+              dlSubdir: ghc-9.4.7-x86_64-unknown-linux
+              dlHash: 6b5b939d20b47475358993210738578071005669500810f28746bbd856ed878b
+            '>= 11': &ghc-947-64-deb11
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-deb11-linux.tar.xz
+              dlSubdir: ghc-9.4.7-x86_64-unknown-linux
+              dlHash: 364d1d0af4289b6492d464e740cc48eda980d2e7904b5d6a0133553adca4c1ab
+            unknown_versioning: *ghc-947-64-deb11
+          Linux_Ubuntu:
+            unknown_versioning: *ghc-947-64-deb10
+            '( >= 16 && < 19 )': *ghc-947-64-deb9
+          Linux_Mint:
+            '< 20': *ghc-947-64-deb9
+            '>= 20': *ghc-947-64-deb10
+          Linux_CentOS:
+            '( >= 7 && < 8 )': &ghc-947-64-centos
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-centos7-linux.tar.xz
+              dlSubdir: ghc-9.4.7-x86_64-unknown-linux
+              dlHash: 20e4314966162d5fe1b29551f09d12e077ef849f41f58a205b9fe66a64da2b1d
+            unknown_versioning: *ghc-947-64-centos
+          Linux_Fedora:
+            '( >= 27 && < 33 )': &ghc-947-64-fedora-27
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-fedora27-linux.tar.xz
+              dlSubdir: ghc-9.4.7-x86_64-unknown-linux
+              dlHash: 216b76b7c6383e6ad9ba82533f323f8550e52893a8b9fa33c7b9dc4201ac766a
+            '>= 33': &ghc-947-64-fedora-33
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-fedora33-linux.tar.xz
+              dlSubdir: ghc-9.4.7-x86_64-unknown-linux
+              dlHash: bc369dc7550b4e45a0c1ebc73b5ea2235e8d0913761f410655a83c21c9ba3e20
+            unknown_versioning: *ghc-947-64-centos
+          Linux_RedHat:
+            '>= 8': *ghc-947-64-fedora-27
+            unknown_versioning: *ghc-947-64-centos
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc-947-64-fedora-27
+          Darwin:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-apple-darwin.tar.xz
+              dlSubdir: ghc-9.4.7-x86_64-apple-darwin
+              dlHash: 2c874dc685cb72b0c4d6f226b795051705a923c25080eeba05d546350474cb1e
+          Windows:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-unknown-mingw32.tar.xz
+              dlSubdir: ghc-9.4.7-x86_64-unknown-mingw32
+              dlHash: 415e6adebd764d278ecfe4ebf656ff96902953b8b669fba7dae2525af6665172
+          Linux_Alpine:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-x86_64-alpine3_12-linux.tar.xz
```

Comment:
Is this dynamic or static?

Reply:
This is dynamic now, as requested in https://gitlab.haskell.org/ghc/ghc/-/issues/23828

Static bindists are quite broken.